### DISCUSSION TITLE
Accidentally nuked my postgres db

### DISCUSSION THREAD

**Discussion by thearchivalone** [firefly-iii/firefly-iii#5566] [Firefly III] [answered]
I have all of the data in my uploads folder. Is there any way to rebuild from that or import them into a new db? Thanks

**Comment by JC5** [comment, marked as answer]
Sorry no :(

You can reupload stuff, but you'll have to manually recreate accounts, rules, etc.

**Reply by thearchivalone** [reply]
I’ve noticed that the data files have png/jpg headers. lf they’re not some weird encrypted format, that’s good enough for my needs. Thanks
